ci/fuzz-test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# NOTE:
+#     run fuzz test after build
+#     required jars: jni, jni-tests, slf4j-api
+
+set -ex
+
+WORKSPACE=${WORKSPACE:-$PWD}
+M2DIR=${M2DIR:-"$HOME/.m2"}
+
+SLF4J_VER=$(mvn help:evaluate -Dexpression=slf4j.version -q -DforceStdout)
+CLASSPATH=${CLASSPATH:-"$WORKSPACE/target/*:$M2DIR/repository/org/slf4j/slf4j-api/$SLF4J_VER/slf4j-api-$SLF4J_VER.jar"}
+
+java -cp "$CLASSPATH" \
+  com.nvidia.spark.rapids.jni.RmmSparkMonteCarlo \
+  --taskMaxMiB=2048 --gpuMiB=3072 --skewed --allocMode=ASYNC
```

Comment:
Did we want to include a random `--seed` argument in the fuzz test script per https://github.com/NVIDIA/spark-rapids-jni/issues/981 ?

Reply:
I changed it so that by default a random seed is used if --seed is not provided. I wanted to make it simpler to run this way by default.